data/modules/Rondel/Rondel.lua

```diff
@@ -0,0 +1,142 @@
+-- Copyright © 2008-2020 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local Engine = require 'Engine'
+local Lang = require 'Lang'
+local Game = require 'Game'
+local Space = require 'Space'
+local cargo = require 'Commodities'
```

Comment:
looks to be unused.

Reply:
Ah, I will take care of that. Thanks.

Reply:
I guess I happen to be using that, actually.